content/docs/command-reference/plots/index.md

```diff
@@ -260,3 +274,53 @@ file:///Users/usr/src/plots/classes.csv.html
 
 > A confusion matrix [template](/doc/command-reference/plots#plot-templates) is
 > predefined in DVC (found in `.dvc/plots/confusion.json`).
+
+### Example: Offline HTML Template
+
+The plots generated by `dvc plots` uses Vega JavaScript libraries, and by
+default these load [online resources](https://vega.github.io/vega/usage/#embed).
+There may be times when you need to produce plots without Internet access, or
+want to customize the plots output to put some extra content, like banners or
+extra text. DVC allows to replace the HTML file that contains the final plots.
```

Comment:
```suggestion
There may be times when you need to produce plots without Internet access.
DVC allows to replace the HTML file that contains the final plots.
```

> or want to customize the plots output to put some extra content, like banners or extra text

The example isn't about that. Maybe mention these other examples in the Description subsection? Will leave suggestion ⏳

Reply:
I don't think we can enumerate all use cases of templates in the description? We _implement_ one of the example ideas, the rest can be done in a similar way.

Reply:
I wasn't suggesting to enumerate every single possible case. My point was that this "Offline HTML Template" example is not about extra HTML content so that sentence just seems unrelated. Could just be removed IMO